package.json

```diff
@@ -103,24 +103,24 @@
     "generate:machine-api": "npx openapi-typescript ./openapi/machine-api.json -o src/lib/machine-api.d.ts",
     "tron:start": "electron-forge start",
     "tron:package": "electron-forge package",
-    "tron:test": "NODE_ENV=development yarn playwright test --config=playwright.electron.config.ts --grep=@electron",
+    "tron:make": "electron-forge make",
+    "tron:publish": "electron-forge publish",
```

Comment:
There are not in `main`, are these required?

Reply:
No that's a conflict resolution oopsie. Only package needs to stay

Reply:
Removed, ty!